subprojects/persistent-cache/src/test/groovy/org/gradle/cache/internal/CacheVersionMappingTest.groovy

```diff
@@ -93,7 +94,7 @@ class CacheVersionMappingTest extends Specification {
     def "throws exception if base version of Gradle version is greater than base version of current Gradle version"() {
         when:
         CacheVersionMapping.introducedIn("1.0")
-            .changedTo(2, GradleVersion.current().nextMajor.version)
+            .changedTo(2, DefaultGradleVersion.current().nextMajorVersion.version)
```

Comment:
Given `GradleVersion.current()` will become an "official" public API, why not keep using the method from `GradleVersion` in this and the other tests?